aten/src/ATen/core/Dimname.cpp

```diff
@@ -54,26 +54,8 @@ Dimname Dimname::fromSymbol(Symbol full_name) {
   if (full_name == kWildcard) {
     return Dimname::wildcard();
   }
-  const std::string delimiter = ".";
-  const std::string str(full_name.toUnqualString());
-  auto it = str.find(delimiter);
-
-  // Check for normal name
-  if (it == std::string::npos) {
-    check_valid_identifier(str);
-    return Dimname(full_name);
-  }
-
-  // Check for tagged name
-  auto second_dot = str.find(delimiter, it + 1);
-  TORCH_CHECK(
-      second_dot == std::string::npos,
-      "Invalid name '", str, "': A tagged name can only contain one '.'");
-  auto untagged_name = str.substr(0, it);
-  auto tag = str.substr(it + 1);
-  check_valid_identifier(untagged_name); 
-  check_valid_identifier(tag);
-  return Dimname(NameType::TAGGED, full_name, Symbol::dimname(untagged_name));
+  check_valid_identifier(full_name.toUnqualString());
+  return Dimname(full_name);
```

Comment:
if we're getting rid of tagged names, should it just be "name" instead of "full_name"?

Reply:
I changed that in the next diff in the stack: https://github.com/pytorch/pytorch/pull/26365.

This one is to just disable creation of tagged names.

Reply:
I can merge the two PRs together (perhaps they make more sense as one unit) if that makes them easier to review.

Reply:
yup, saw it.